src/main/java/in/ashwanthkumar/gocd/slack/ruleset/Rules.java

```diff
@@ -92,6 +93,19 @@ public Rules setGoServerHost(String goServerHost) {
         return this;
     }
 
+
+    public String getGoAPIServerHost() {
+        if (goAPIServerHost != null) {
```

Comment:
Will it help if we can search for `StringUtils.IsNotBlank` instead of just a null check?